pire/scanners/common.h

```diff
@@ -61,6 +64,36 @@ namespace Pire {
 			if ((type != 0 && type != Type) || (hdrsize != 0 && HdrSize != hdrsize))
 				throw Error("Serialized regexp incompatible with your system");
 		}
+
+		void Pack(yostream* s)
+		{
+			char buf[PACKED_HEADERS * 8 * 4];
```

Comment:
What is the meaning of 4? Could you replace 8 and 4 with named constants, please?